avocado/utils/process.py

```diff
@@ -158,29 +158,6 @@ def kill_process_by_pattern(pattern):
         logging.info("Succeed to run '%s'.", cmd)
 
 
-def process_in_ptree_is_defunct(ppid):
```

Comment:
This function is used in `aexpect`. Aexpect does not depend on avocado so it might be safe to remove it, but I think it could stay as some tests might use it.